app/models/concerns/authorizable.rb

```diff
@@ -23,14 +23,15 @@ def permissions
     ADOPTER_PERMISSIONS + %i[
       manage_pets
       manage_tasks
+      view_organization_dashboard
```

Comment:
So I think we will end up with a separate dashboard for Org Staff, and another for Adopters and Fosterers. So, we can remove adopter ability to view the org dashboard, which is for staff.